cime_config/machines/config_machines.xml

```diff
@@ -3240,10 +3240,8 @@
     <BATCH_SYSTEM>pbspro</BATCH_SYSTEM>
     <SUPPORTED_BY>e3sm</SUPPORTED_BY>
     <MAX_TASKS_PER_NODE>208</MAX_TASKS_PER_NODE>
-    <MAX_TASKS_PER_NODE compiler="oneapi-ifx">208</MAX_TASKS_PER_NODE>
-    <MAX_TASKS_PER_NODE compiler="oneapi-ifxgpu">104</MAX_TASKS_PER_NODE>
+    <MAX_TASKS_PER_NODE compiler="oneapi-ifxgpu">96</MAX_TASKS_PER_NODE>
```

Comment:
Why is this changed for a specific compiler?

Reply:
For now, setting it to 2x the number of MPI tasks per node (48): 96 threads/node for GPU builds.
On my to-do is to look at best combinations of tasks and threads for each compiler build.